projects/fizzbuzz/python/b-cornett/README.md

```diff
@@ -0,0 +1,9 @@
+Instructions for running the python program for FizzBuzz
+
+1.) Verify Python is installed on your machine.
+2.) Verify your PATH variable is set correctly to run the 'python' command
+3.) Clone the project, go to the /python/b-cornett folder. Run the following:
+
```

Comment:
You could possibly use the ``` on the line above and the line below your code block to have the GitHub markdown show as a code block when a user views it in GitHub via a web browser.

Reply:
@TimParrish Thanks for the input Tim! I added the 3 back-ticks above and below the Python command.